### ISSUE TITLE
Selecting Specialty/Data Sets

### ISSUE THREAD

**Issue by BrandoCC** [canadian-federation-of-medical-students/CFMS-Issue-Tracker#32]
This would something that would be considered on an additional contract. I believe that when selecting the specialty and the data sets it would be more user friendly if instead of the columns, the options were displayed as a big pop up window. Within the window all the options would be displayed as one table (ex: 5X5 or whatever combination that works) for each option. Then once the options are selected they will appear on the left bar. This way I it would clean up the options on the left side! Additionally, I think we should have the option to select the view in the left column above the selecting a year and have this as the first option. Then have the user select the year. I think these changes would make it much more user friendly as they may be overwhelmed with all the different options available.

**Comment by Jrschellenberg** [comment]
We can discuss this in more detail to figure out if we should want to invest the time to Design the application this way. However, I will say that Raj, Jt, and myself thought about approaching initially to design the application this way, however, we decided to not go forth with this.


The reason is that typically in a Design, the more windows, steps, a user has to take, it overall makes the application harder to learn and hurts the User experience.

With that being said, if we spent time to help the blank slate (first time user uses app) of the application, then we may be able to approach it from this design perspective!
